models/certificateTemplates.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    name: "certificateTemplates",
+    schema: {
+      templateUrl: {
+        type : String,
+        required : true
+      },
+      issuer: Object,
+      status: {
```

Comment:
@VISHNUDAS-tunerlabs Add default value in status

Reply:
@priyanka-TL added default : "ACTIVE"